packages/teams-js/src/public/runtime.ts

```diff
@@ -45,6 +45,9 @@ interface IRuntimeV1 extends IBaseRuntime {
     readonly geoLocation?: {
       readonly map?: {};
     };
+    readonly hostEntity?: {
+      readonly tab?: {};
+    };
```

Comment:
Since it's optional, I don't think it's harmful to add this to the older version of the runtime interface in this library, but unless you have any plans to go to older host sdk versions and add it there, there's no reason to add it to older runtime interface versions.

Reply:
Removed